### DISCUSSION TITLE
Educational benefits not yet started

### DISCUSSION THREAD

**Discussion by mohammeduvez29** [community/community#134848] [GitHub Education]
### Hi everyone,

Many days passed since approval, but still educational benifits has not been enabled

**Comment by elsycharles** [comment]
Hi, 
please re-visit the website [https://education.github.com](https://education.github.com) 

- check if your school is recognized by GitHub Check to see if your school is in [GitHub Campus](https://education.github.com/schools) or [Microsoft Education Program](https://www.microsoft.com/en-in/education). If not, then you may need to contact GitHub Support for additional support.

-  Use up to date documents/information. Provide the most up to date information and documents otherwise you might be rejected if it is too outdated.

-  Ensure your GitHub profile matches. Make sure all information such as your name or location both match your profile and documents.

-  Make sure your school email is linked. You should have your school email verified in your GitHub account.

-  Don't use a VPN. GitHub may detect you using a VPN and automatically deny you access. Try applying without any VPN.

You can read "[Why wasn't my application to Global Campus for students approved?](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved)" and the "[Student Developer Pack Application & FAQs ➡️ Start Here #17814](https://github.com/orgs/community/discussions/17814)" discussion for more information about Global Campus rejections.

As a reminder you do not need to have a Student ID to get the GitHub Student Developer Pack, for more information see "[How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)"

If none of this helps, you can contact GitHub for support at [Get help with GitHub Education](https://support.github.com/contact/education).

I hope this helps you with your question. Once resolved, you can mark this comment as the answer.

**Reply by mohammeduvez29** [reply]
Thank you so much for your response. Now the benefits has been activated
